supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -101,13 +101,17 @@ Workaround: Rerun the deployment command, or use a local container image registr
 [discrete]
 ==== Technology Preview
 
-Use present tense in the following format:
+Use present tense in the following format, where `<TechPreviewName>` is the name of your product, component or feature:
+
+// The following is based on RHEL 9 snip_techpreview.adoc
 ----
-In <release>, a Technology Preview is available for <feature explanation>.
+<TechPreviewName> is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
```

Comment:
To follow our replaceable value guidance can you change `<TechPreviewName>` to `<tech_preview_name>` here and above?

Reply:
Fixed.